react/tic-tac-toe/src/redux/Matches/actions.js

```diff
@@ -0,0 +1,22 @@
+import MatchesService from '../../services/MatchesService';
+
+export const actions = {
+  GET_MATCHES: '@@MATCHES/GET_MATCHES',
+  LOAD_MATCHES: '@@MATCHES/LOAD_MATCHES'
+};
+
+const loadMatches = data => ({
+  type: actions.LOAD_MATCHES,
+  payload: { data }
+});
+
+const getMatches = () => ({
+  type: actions.GET_MATCHES
+});
+
+const fetchMatches = () => async (dispatch) => {
+  dispatch(getMatches());
+  await MatchesService.getMatches().then(({ data }) => dispatch(loadMatches(data)));
```

Comment:
Using the `async/await` replace the `Promise/then` pattern.
```js
const response = await MatchesService.getMatches();
```
You should probably check (not necessary in this case) if `response.data` exists and dispatch if so.
In a real project you would dispatch a success or failure action depending on the response